site/Makefile

```diff
@@ -11,6 +11,10 @@ generate: $(TOOLBIN)/crdoc $(TOOLBIN)/protoc $(TOOLBIN)/protoc-gen-doc
 	PATH=$(TOOLBIN) protoc -I=$(CONNECTORS_PROTO) --doc_out=./docs/reference/ --doc_opt=./templates/proto/protoc.tmpl,connectors.md $(sort $(wildcard ./$(CONNECTORS_PROTO)/*.proto))
 	go mod tidy
 
+generate-openapi-doc:
```

Comment:
Missing PHONY. 
Also, `generate` should probably call it too (?)

Reply:
fixed. Please check.